Community/kavSimulations/devices/kav_battery.device.json

```diff
@@ -16,6 +16,11 @@
       "isI2C": false
     },
     "MessageTypes": [
+      {
+        "id": -1,
```

Comment:
This is a built in message type. Shouldn't this be 255 instead? Does it make sense to have this visible to the user?

Reply:
I don't see any drawback to use this messageID.

Adding another one means just adding another messageID which does the same.